<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>szukajKlienta</class>
 <widget class="QDialog" name="szukajKlienta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="szukajButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrocButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QLineEdit" name="NumerLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="odCheckBox">
      <property name="text">
       <string>Od daty</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="DocheckBox">
      <property name="text">
       <string>Do daty</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateEdit" name="doDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDateEdit" name="odDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QCheckBox" name="numerCheckBox">
      <property name="text">
       <string>Numer telefonu klienta</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
